src/styles/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url('/src/images/star.svg') no-repeat center;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url('/src/images/star-active.svg');
+  }
```

Comment:
Consider adding the code for modifiers from the previous task to keep them working